mayadata-io/gui-automation
# Readme files with Selenium test cases for Authentication and Cluster

#### Authentication: gui-local-auth-check.md

* To verify Administrator login
* To verify signup functionality in Local Auth
* To verify the Change password functionality for local auth account
* To verify error message is shown if wrong password is provided
* To verify error message is shown if wrong email ID is provided
* Password for local authentication should be alpha numeric supported
* To verify unique Email ID for each user

#### Cluster: gui-self-cluster-check.md

* To verify self connected cluster is shown for Admin user
* To verify cluster name should not be less than 6 character and should not contain any special character
